test/util/component_usage_migrator_test.dart

```diff
@@ -1465,6 +1466,147 @@ void yieldRemoveChildPatchTests() {
   });
 }
 
+void yieldAddChildPatchTests() {
+  test('when it is the only child', () async {
```

Comment:
#nit I think this test description could be clarified a little
```suggestion
  test('when there are no existing children', () async {
```